### DISCUSSION TITLE
How to alias a column during a select?

### DISCUSSION THREAD

**Discussion by narigama** [SeaQL/sea-query#778] [Q&A] [answered]
I'm looking to generate `SELECT table.foo as bar` style syntax to make handling a poorly named column more ergonomic. Whilst I can generate `table.foo`, I can't find anything to handle the aliasing.

**Comment by seijikun** [comment, marked as answer]
This should work using something like:
```rust
let mut query = Query::select();
   query.expr_as(
       Expr::col(db::Table::Column::Foo),
       Alias::new("bar")
   )
```

**Reply by seijikun** [reply]
If it helped you, can you please also mark the answer?
